src/App.vue

```diff
@@ -1,11 +1,70 @@
 <template>
   <div id="app">
-    
+    <h1>ToDoリスト</h1>
+    <div>
+      <div>
+        <label><input type="radio" id="all" v-model="radio" value="すべて">すべて</label>
+        <label><input type="radio" id="working" v-model="radio" value="作業中">作業中</label>
+        <label><input type="radio" id="complete" v-model="radio" value="完了">完了</label>
+      </div>
+      <table>
+        <tr>
+          <th>ID</th>
+          <th>コメント</th>
+          <th>状態</th>
+        </tr>
+        <tr v-for="(todo, index) in $store.state.todos" :key="index" v-show="checkState(todo)">
+          <td>{{ index }}</td>
+          <td>{{ todo['Comment'] }}</td>
+          <td><button type="button" @click="changeState">{{ todo['State'] }}</button></td>
+          <td><button type="button" @click="deleteTodo">削除</button></td>
+        </tr>
+      </table>
+      <h2>新規タスクの追加</h2>
+      <form>
+        <input type="text" v-model="comment">
+        <button type="button" @click="addTodo(comment); deleteText()">追加</button>
+      </form>
+    </div>
   </div>
 </template>
 
 <script>
+import { todos } from './store/index.js'
+import { mapMutations } from 'vuex'
 
+export default {
+  name: 'App',
+  data: () => {
+    return {
+      todos,
+      // 下記2つの定義値は一時的にしか使用にないためvuexで値を持たせる必要がないと判断した
+      radio: 'すべて',
+      comment: ''
+    }
+  },
+  methods: {
+    // ストアに定義してあるtodosを使う関数のみマッピングした
+    // deleteText()とcheckState(todo)はtodosを使わないためマッピングの必要がないと判断した
+    ...mapMutations(['addTodo', 'deleteTodo', 'changeState']),
+    deleteText(){
+      this.comment = ''
+    },
+    // ラジオボタンに応じて全てのtodoの状態を確認してソート
+    checkState(todo){
+      if(this.radio === 'すべて'){
```

Comment:
- `{}`の両外側
- ifの後ろの`()`の両外側
上記にはスペースをあけるようにしましょう。
他の場所でも同様です。
```suggestion
      if (this.radio === 'すべて') {
```
自動整形を使うといいでしょう。